### PR TITLE
Clarify registry review process.

### LINKED ISSUE #170: Consider clarifying criteria by which registry table changes are proposed, approved, and incorporated
Copied over from https://github.com/w3c/transitions/issues/323#issuecomment-812688759:

> Please consider, for future updates, expanding on the 5th bullet in Registration Entry Requirements, since it only mentions how to propose but doesn't say how propose changes get approved.

This was also raised by @aboba in the pull request that established the registry: https://github.com/w3c/webcodecs/pull/149#issuecomment-801228773

### PR DESCRIPTION
**Pull request by chcunningham**
Fixes #170.

### CODE REVIEW

File: codec_registry.src.html [hunk 1]
```diff
@@ -75,7 +75,10 @@
 4. Candidate entries must be announced by filing an issue in the
     [WebCodecs GitHub issue tracker](https://github.com/w3c/webcodecs/issues/)
     so they can be discussed and evaluated for compliance before being added to
-    the registry.
+    the registry. If registry editors reach consesnus to accept the candidate,
```
Comment (tidoust) [inline codec_registry.src.html:78]:
```suggestion
    the registry. If registry editors reach consensus to accept the candidate,
```

**Review by tidoust — COMMENTED** [review]
I understand "registry editors" to mean Chris, Paul and Bernard. That's a fantastic set of people, but why isn't the [custodian](https://www.w3.org/Consortium/Process/Drafts/#custodian) the Media Working Group, in other words the entity that will be chartered to develop and maintain the registry?

Note that, even if you switch to the Media Working Group, mention of registry editors for review and merging of the pull request seems totally appropriate.

**Review by aboba — APPROVED** [review]

**Comment by chcunningham** [comment]
@tidoust can you propose some a rewording that mentions the WG? To me it seems weird, as it really is the specific editors that should conduct the review.

**Comment by chcunningham** [comment]
> @tidoust can you propose some a rewording that mentions the WG? To me it seems weird, as it really is the specific editors that should conduct the review.

I'll go ahead and merge. Happy to amend if you come back with suggestions.

**Pull request merged**